api/topics/monitor/retrieve_batch_list.go

```diff
@@ -0,0 +1,10 @@
+package monitor
+
+// RetrieveBatchList returns a list of the current batches
+func (ctl BatchController) RetrieveBatchList() (map[string]string, error) {
+	list, err := ctl.rdb.HGetAll("batch").Result()
+	if err != nil {
+		return nil, err
+	}
+	return list, nil
```

Comment:
Would returning `ctl.rdb.HGetAll("batch").Result()` be equivalent here? Because we don't process the error at all